### PR TITLE
Feat: added List and Modal component in CL

### PR DESCRIPTION
**Pull request by sakleprajwal**
Added List and Modal component in spacUi

image

image

image

### CODE REVIEW

File: component styles/List.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    padding: 0.5rem ;
+    border-radius: 0.5rem;
+    list-style: none;
+}
+.notification-pill{
+    display: flex;
+    background-color: var(--ternary-color-shadow);
+    gap: 1rem;
+    padding: 1rem 1.5rem;
+    box-shadow: 1px 2px 25px 1px var(--lightest-gray);
```
Comment (therajatg) [inline component styles/List.css]: You can have box-shadow as variable. So that the same can be reused.

**Review by therajatg — APPROVED** [review]
Everything else looks good

File: component styles/Modal.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    top: 0;
+    width: 100%; /* Full width */
+    height: 100%; /* Full height */
+    overflow: auto; /* Enable scroll if needed */
+    background-color: rgb(0,0,0); /* Fallback color */
+    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+  }
+  
+  /* Modal Content */
+  .modal-content {
```
Comment (arundhatij12) [inline component styles/Modal.css:16]: try using color variables

File: component styles/Modal.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    z-index: 1; /* Sit on top */
+    padding-top: 100px; /* Location of the box */
+    left: 0;
+    top: 0;
+    width: 100%; /* Full width */
+    height: 100%; /* Full height */
+    overflow: auto; /* Enable scroll if needed */
+    background-color: rgb(0,0,0); /* Fallback color */
+    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+  }
```
Comment (Mahi24-prog) [inline component styles/Modal.css:13]: Use color variable

File: component styles/Modal.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    width: 80%;
+  }
+  
+  /* The Close Button */
+  .close {
+    color: #aaaaaa;
+    float: right;
+    font-size: 28px;
+    font-weight: bold;
+  }
```
Comment (Mahi24-prog) [inline component styles/Modal.css:29]: use predefined class for font related style

**Review by Mahi24-prog — APPROVED** [review]
Looks good go through suggestion once

**Pull request merged**